Items/Armor/Wulfrum/WulfrumSet.cs

```diff
@@ -84,7 +87,7 @@ private void ActivateSetBonus(Terraria.On_Player.orig_KeyDoubleTap orig, Player
                 else if (player.HasItem(ModContent.ItemType<WulfrumMetalScrap>()))
                 {
                     player.ConsumeItem(ModContent.ItemType<WulfrumMetalScrap>());
-                    //I Thiiiinnnk there's no need to add mp syncing packets sicne cooldowns get auto synced right.
+                    //I Thiiiinnnk there's no need to add mp syncing packets since cooldowns get auto synced right.
```

Comment:
FYi fixing typos and formatting and stuff should usually be kept out of PRs like these. I'd be fine reviewing and merging a PR dedicated to such tasks.